transformers/component_tests/queries/get_clip_query_test.go

```diff
@@ -0,0 +1,66 @@
+package queries
+
+import (
+	"math/rand"
+	"strconv"
+
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/component_tests/queries/test_helpers"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/libraries/shared/repository"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres/repositories"
+	"github.com/makerdao/vulcanizedb/pkg/fakes"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Single clip view", func() {
+	var (
+		blockOne, timestampOne int
+		addressId              int64
+		contractAddress        = fakes.FakeAddress.Hex()
+		fakeSaleId             = rand.Int()
+		headerOne              core.Header
+		headerRepo             datastore.HeaderRepository
+	)
+
+	BeforeEach(func() {
+		test_config.CleanTestDB(db)
+		headerRepo = repositories.NewHeaderRepository(db)
+
+		blockOne = rand.Int()
+		timestampOne = int(rand.Int31())
+		headerOne = createHeader(blockOne, timestampOne, headerRepo)
+
+		clipKickLog := test_data.CreateTestLog(headerOne.Id, db)
+
+		var addressErr error
+		addressId, addressErr = repository.GetOrCreateAddress(db, contractAddress)
+		Expect(addressErr).NotTo(HaveOccurred())
+
+		clipKickEvent := test_data.ClipKickModel()
+		clipKickEvent.ColumnValues[event.HeaderFK] = headerOne.Id
+		clipKickEvent.ColumnValues[event.LogFK] = clipKickLog.ID
+		clipKickEvent.ColumnValues[event.AddressFK] = addressId
+		clipKickEvent.ColumnValues[constants.SaleIDColumn] = strconv.Itoa(fakeSaleId)
+		clipKickErr := event.PersistModels([]event.InsertionModel{clipKickEvent}, db)
+		Expect(clipKickErr).NotTo(HaveOccurred())
+	})
+
+	Context("get_clip_with_address", func() {
+		It("gets only the specified clip", func() {
```

Comment:
Seems like this test is missing a second clip with the same sale ID from another address. Should we create a second clip as part of this test and verify that we're only retrieving the one we expect?